### PR TITLE
Release 1.0

### PR DESCRIPTION
**Pull request by ronijaakkola**
## Description
Updating Roadmap, What's new section and adding changelogs for the 1.0 release.

## How Has This Been Tested?
Tested by running the documentation site locally.

### CODE REVIEW

File: site/docs/about/roadmap.mdx [hunk 1]
```diff
@@ -17,7 +17,9 @@ import LargeParagraph from "../../src/components/LargeParagraph";
 
 - **HDS Beta release 29.10.2020**
 
-For previously released features, see changelogs in the [What is new section](/about/new)
+- **HDS 1.0 release 24.04.2021**
```
Comment (laurakarhu) [inline site/docs/about/roadmap.mdx:20]: 29.4.2021 (Correct date and HDS date format)

**Review by laurakarhu — CHANGES_REQUESTED** [review]
Date needs fixing, otherwise all good.

File: site/docs/about/new.mdx [hunk 2]
```diff
@@ -13,6 +13,14 @@ import Link from "../../src/components/Link";
     Here you will find summarized patch notes of major releases of HDS. For full patch notes for each release, please refer to the <Link href="https://github.com/City-of-Helsinki/helsinki-design-system/releases" external>GitHub releases</Link>.
 </LargeParagraph>
 
+<p><strong>1.0</strong> - <em>29.4.2021</em> - <Link href="https://github.com/City-of-Helsinki/helsinki-design-system/releases/tag/v1.0" external>Release notes</Link></p>
+
+- **Changed**: Documentation and component statuses updated for the 1.0 version
+- **Fixed**: Small fixes and improvements for multiple components
+- **Removed**: Previously deprected Dropdown component
```
Comment (minevala) [inline site/docs/about/new.mdx:20]: deprecated

**Review by minevala — COMMENTED** [review]
A language tool finding.

**Review by minevala — APPROVED** [review]

**Review by laurakarhu — APPROVED** [review]
Good job!

**Pull request merged**